README.md

```diff
@@ -1,18 +1,44 @@
 # Supabase Edge Runtime
 
-A web server based on [Deno](https://deno.land) runtime, capable of running JavaScript, TypeScript, and WASM services.
+A Javascript runtime based on [Deno](https://deno.land), capable of running JavaScript, TypeScript, and WASM services.
```

Comment:
"JavaScript runtime capable of running JavaScript" is a bit redundant. Maybe just a "runtime"? It's technically a web server though.